src/core/utils/findAllElementsOfType.ts

```diff
@@ -0,0 +1,14 @@
+import { getChildrenByType } from './getChildrenByType';
+import { recursiveMap } from './recursiveMap';
+
+export const findAllElementsOfType = (
+  children: React.ReactNode,
+  types: string[],
+) => {
```

Comment:
Adding function documentation / JSDoc would add to the understandability of the code at a glance.